h/static/extension/config.js

```diff
@@ -1,8 +0,0 @@
-function hypothesisConfig() {
-  // Pages on our site can include a meta tag to trigger specific behaviour
-  // when the extension loads.
-  var hypothesisIntent = document.querySelector('[name="hypothesis-intent"]');
```

Comment:
Gah. I'd moved that function for detecting the first-run use case to config.js in the direct-linking branch. But when I separated out 291-detect-existing-h-client into a separate branch where config.js isn't present, I hadn't moved the code back to `annotator/main.js`.